DEV-2459: Review and Improvements (#643)
cloudposse/docs: docs/layers/identity/centralized-terraform-access.mdx

account. These are redundant with AWS Team Roles but are useful for quickly accessing a given account with limited
permission.

> With AWS IAM Identity Center, you can connect directly to the given account from the single sign on page with the Permission Set you need. That's great for web console usage, but it isn't great for locally applying Terraform! With Terraform, we want to apply across many accounts, where each account has its own role. We don't want to have to assume a new role each time we change accounts. So to solve this, we have a specific Permission Set that can assume an AWS Team in the identity account. When applying Terraform, we assume that "team" via SSO, and when accessing the web console, we use the Permission Set directly.

Reviewer: ```suggestion
With AWS IAM Identity Center, you can connect directly to a given account from the single sign-on page with the required `PermissionSet`. This is great for web console usage but not ideal for locally applying Terraform. With Terraform, we need to apply changes across many accounts, where each account has its own role. We don't want to assume a new role each time we change accounts. To solve this, we have a specific Permission Set that can assume an AWS team role in the identity account. When applying Terraform, we assume that "team" role via SSO, and when accessing the web console, we use the Permission Set directly.
```

> With AWS IAM Identity Center, you can connect directly to a given account from the single sign-on page with the required `PermissionSet`. This is great for web console usage but not ideal for locally applying Terraform. With Terraform, we need to apply changes across many accounts, where each account has its own role. We don't want to assume a new role each time we change accounts. To solve this, we have a specific Permission Set that can assume an AWS team role in the identity account. When applying Terraform, we assume that "team" role via SSO, and when accessing the web console, we use the Permission Set directly.

Reviewer: ```suggestion
With AWS IAM Identity Center, you can directly access a specific account from the single sign-on page, which will automatically assume the role with the associated `PermissionSet`. This is great for web console usage but not ideal for locally applying Terraform. With Terraform, we need to apply changes across many accounts, where each account has its own role. We don't want to assume a new role each time we change accounts. To solve this, we have a specific Permission Set that can assume an AWS team role in the identity account. When applying Terraform, we assume that "team" role via SSO, and when accessing the web console, we use the Permission Set directly.
```
